src/lib.rs

```diff
@@ -1,39 +1,41 @@
 use std::vec::Vec;
 
-extern "C" {
-    fn ethereum_useGas(amount: u64);
-    fn ethereum_getGasLeft() -> u64;
-    fn ethereum_getAddress(resultOffset: *const u32);
-    fn ethereum_getBalance(addressOffset: *const u32, resultOffset: *const u32);
-    fn ethereum_getBlockCoinbase(resultOffset: *const u32);
-    fn ethereum_getBlockDifficulty(resultOffset: *const u32);
-    fn ethereum_getBlockGasLimit() -> u64;
-    fn ethereum_getBlockHash(number: u64, resultOffset: *const u32) -> u32;
-    fn ethereum_getBlockNumber() -> u64;
-    fn ethereum_getBlockTimestamp() -> u64;
-    fn ethereum_getTxGasPrice(valueOffset: *const u32);
-    fn ethereum_getTxOrigin(resultOffset: *const u32);
-    fn ethereum_log(dataOffset: *const u32, length: u32, numberOfTopics: u32, topic1: *const u32, topic2: *const u32, topic3: *const u32, topic4: *const u32);
-    fn ethereum_call(gas: u64, addressOffset: *const u32, valueOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callCode(gas: u64, addressOffset: *const u32, valueOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callDelegate(gas: u64, addressOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_callStatic(gas: u64, addressOffset: *const u32, dataOffset: *const u32, dataLength: u32) -> u32;
-    fn ethereum_create(valueOffset: *const u32, dataOffset: *const u32, dataLength: u32, resultOffset: *const u32) -> u32;
-    fn ethereum_returnDataCopy(resultOffset: *const u32, dataOffset: u32, length: u32);
-    fn ethereum_getReturnDataSize() -> u32;
-    fn ethereum_finish(dataOffset: *const u32, length: u32) -> !;
-    fn ethereum_revert(dataOffset: *const u32, length: u32) -> !;
-    fn ethereum_callDataCopy(resultOffset: *const u32, dataOffset: u32, length: u32);
-    fn ethereum_getCallDataSize() -> u32;
-    fn ethereum_getCaller(resultOffset: *const u32);
-    fn ethereum_getCallValue(resultOffset: *const u32);
-    fn ethereum_codeCopy(resultOffset: *const u32, codeOffset: u32, length: u32);
-    fn ethereum_getCodeSize() -> u32;
-    fn ethereum_externalCodeCopy(addressOffset: *const u32, resultOffset: *const u32, codeOffset: u32, length: u32);
-    fn ethereum_getExternalCodeSize(addressOfset: *const u32) -> u32;
-    fn ethereum_storageLoad(keyOffset: *const u32, resultOffset: *const u32);
-    fn ethereum_storageStore(keyOffset: *const u32, valueOffset: *const u32);
-    fn ethereum_selfDestruct(addressOffset: *const u32) -> !;
+mod native {
```

Comment:
Perhaps call it FFI or something along the lines of `host_interface`

Reply:
Isn't `native` fitting?